clang/lib/ASTMatchers/ASTMatchersInternal.cpp

```diff
@@ -36,6 +36,7 @@
 #include <cstddef>
 #include <optional>
 #include <string>
+#include <unordered_set>
```

Comment:
Would `llvm::DenseSet` make more sense?

https://llvm.org/docs/ProgrammersManual.html#llvm-adt-denseset-h

Reply:
very good sugguestion! updated in f3f08877f08c